packages/fxa-payments-server/package.json

```diff
@@ -7,7 +7,7 @@
     "lint:eslint": "eslint .",
     "lint:tsc": "tsc --noEmit",
     "start": "node server/bin/fxa-payments-server.js",
-    "start-dev": "react-scripts start",
+    "start-dev": "concurrently -k \"PROXY_STATIC_RESOURCES_FROM='http://127.0.0.1:3032' node server/bin/fxa-payments-server.js\" \"react-scripts start\"",
```

Comment:
Only nit I might have here is that this could be a boolean switch like `PROXY_STATIC_RESOURCES=1` and stick the actual URL in `server/config/index.js` as a default value. But, meh, just a small nit

Reply:
Also, I think I'm going to just merge this - we can consider tweaking this (or not) in a follow up for further settings work